plugin/web/sharedComponents/Icon/icons.ts

```diff
@@ -76,8 +78,7 @@ import { ReactComponent as GearWheel } from "./Icons/gearWheel.svg";
 import { ReactComponent as ExternalLink } from "./Icons/externalLink.svg";
 
 // Ant Design Icons
-import { HolderOutlined } from "@ant-design/icons";
-import { PlayCircleFilled } from "@ant-design/icons";
+import { HolderOutlined, PlayCircleFilled } from "@ant-design/icons";
 
 export { InboxOutlined, DownloadOutlined } from "@ant-design/icons";
```

Comment:
Can you import them all at once since they are all coming from @ant-design/icons?

Reply:
one of the import and the other is export so they should keep like this